### PR TITLE
WSSQL-161 Fixed linking with esp generated services

### PR DESCRIPTION
**Pull request by Michael-Gardner**
Signed-off-by: Michael Gardner <michael.gardner@lexisnexis.com>

### CODE REVIEW

**Comment by Michael-Gardner** [comment]
@rpastrana Please take a look.  Fixes the linking issues we were having with 6.4.0 builds.

File: esp/services/ws_sql/CMakeLists.txt [hunk 1]
```diff
@@ -52,10 +53,15 @@ ADD_CUSTOM_COMMAND(
 )
 
 SET(    SRCS
-        ${ESPSCM_GENERATED_DIR}/ws_sql_esp.cpp
         ${CMAKE_CURRENT_SOURCE_DIR}/ws_sqlPlugin.cpp
         ${CMAKE_CURRENT_SOURCE_DIR}/ws_sqlService.cpp
-        ${HPCC_SQL_PARSER_LOCATION}/HPCCFileCache.cpp
+		${ESPSCM_GENERATED_DIR}/common_esp.cpp
+        ${ESPSCM_GENERATED_DIR}/ws_sql_esp.cpp
+		${ESPSCM_GENERATED_DIR}/ws_workunits_esp.cpp
```
Comment (rpastrana) [inline esp/services/ws_sql/CMakeLists.txt]: minor, but indention seems off here

File: esp/services/ws_sql/CMakeLists.txt [hunk 2]
```diff
@@ -100,24 +107,24 @@ INCLUDE_DIRECTORIES (
...
-        ws_ecl
         ws_dfu
         jlib
         xmllib
         esphttp
         securesocket
         dalibase
         libantlr3c
-    )
+		wuwebview
```
Comment (rpastrana) [inline esp/services/ws_sql/CMakeLists.txt]: indention here's off too

**Comment by Michael-Gardner** [comment]
@rpastrana Fixed all the indentation

**Comment by rpastrana** [comment]
looks good!

**Pull request merged**